pandas/tests/frame/test_api.py

```diff
@@ -261,8 +263,27 @@ def test_itertuples(self, float_frame):
         df3 = DataFrame({"f" + str(i): [i] for i in range(1024)})
         # will raise SyntaxError if trying to create namedtuple
         tup3 = next(df3.itertuples())
-        assert not hasattr(tup3, "_fields")
         assert isinstance(tup3, tuple)
+        if PY37:
+            assert hasattr(tup3, "_fields")
```

Comment:
This test has changed as we will return named tuples always for python >= 3.7 now.